tests/UnifiedSpecTests/Operation.php

```diff
@@ -541,6 +541,35 @@ private function executeForCollection(Collection $collection)
                     array_diff_key($args, ['to' => 1]),
                 );
 
+            case 'createSearchIndex':
+                $options = [];
+                if (isset($args['model']->name)) {
+                    assertIsString($args['model']->name);
+                    $options['name'] = $args['model']->name;
+                }
+
+                return $collection->createSearchIndex($args['model']->definition, $options);
```

Comment:
Should this operation assert that `definition` is a stdClass instance? That would be consistent with what you do below for `updateSearchIndex`:

```php
assertInstanceOf(stdClass::class, $args['definition']);
```

Reply:
Added